### ISSUE TITLE
Conflicting results for test 6 "Request information about a menu item" (Action Menu Button Example Using aria-activedescendant, V24.08.12)

### ISSUE THREAD

**Issue by IsaDC** [w3c/aria-at#1109]
## Description of Behavior

See results below.

## Test Setup

- Test File: [test-16-reqInfoAboutMenuItem-voiceover_macos.collected.html](https://aria-at.netlify.app/tests/menu-button-actions-active-descendant/test-16-reqInfoAboutMenuItem-voiceover_macos.collected.html)
- AT: VoiceOver for macOS (Version 14.6.1)
- Browser: Safari (Version 17)

## Review Conflicts for "Request information about a menu item"

1.  ### Assertion Results for "Control+Option+F3" Command and "Name of the menu, 'Actions', is conveyed" Assertion
    
    *   Tester jha11y recorded output "menu is in the VoiceOver cursor You are currently on a menu." and marked assertion as failing.
    *   Tester IsaDC recorded output "Actions menu is in the VoiceOver cursor" and marked assertion as passing.
2.  ### Assertion Results for "Control+Option+F4" Command and "Name of the menu, 'Actions', is conveyed" Assertion
    
    *   Tester jha11y recorded output "menu has keyboard focus You are currently on a menu." and marked assertion as failing.
    *   Tester IsaDC recorded output "Actions menu has keyboard focus" and marked assertion as passing.

**Comment by IsaDC** [comment]
Relates to #1104 
CC @jha11y  @mcking65  @jscholes

**Comment by mcking65** [comment]
Resolved
